runtime/vm/jfr.cpp

```diff
@@ -965,16 +965,17 @@ jfrCPULoad(J9VMThread *currentThread)
 				jfrEvent->jvmSystem = 0;
 			} else {
 				int64_t timeDelta = currentTime - jfrState->prevProcTimestamp;
-				jfrEvent->jvmUser = OMR_MIN((currentProcCPUTimes._userTime - jfrState->prevProcCPUTimes._userTime) / ((double)numberOfCpus * timeDelta), 1.0);
-				jfrEvent->jvmSystem = OMR_MIN((currentProcCPUTimes._systemTime - jfrState->prevProcCPUTimes._systemTime) / ((double)numberOfCpus * timeDelta), 1.0);
+				float timeElapsedAllCores = (float)numberOfCpus * timeDelta;
+				jfrEvent->jvmUser = OMR_MIN((currentProcCPUTimes._userTime - jfrState->prevProcCPUTimes._userTime) / timeElapsedAllCores, 1.0f);
+				jfrEvent->jvmSystem = OMR_MIN((currentProcCPUTimes._systemTime - jfrState->prevProcCPUTimes._systemTime) / timeElapsedAllCores, 1.0f);
```

Comment:
That's better, but this still (normally) evaluates 
`(currentProcCPUTimes._userTime - jfrState->prevProcCPUTimes._userTime) / timeElapsedAllCores`
twice. Suggest:
```c
				float userTime = (currentProcCPUTimes._userTime - jfrState->prevProcCPUTimes._userTime) / timeElapsedAllCores;
				float systemTime = (currentProcCPUTimes._systemTime - jfrState->prevProcCPUTimes._systemTime) / timeElapsedAllCores;

				jfrEvent->jvmUser = OMR_MIN(userTime, 1.0f);
				jfrEvent->jvmSystem = OMR_MIN(systemTime, 1.0f);
```
A similar issue exists on line 978.